lib/rules/web/admin_console/4.12/operator_hub.xyaml

```diff
@@ -35,11 +35,19 @@ create_cr_infinispancluster:
 create_operand:
   params:
     tab_name: <operand_kind>
-  action: goto_csv_detail_page
   action: click_tab
   action: click_create_operand_button
   action: wait_box_loaded
+  action: switch_to_yaml_view
   action: submit_changes
+create_operand_anzo:
+  params:
+    operand_kind: Anzo
+  action: create_operand
+create_operand_cluster:
+  params:
+    operand_kind: Cluster
+  action: create_operand  
```

Comment:
What is the purpose of 'create_operand_anzo' & 'create_operand_cluster'? Seems the changes will not impact cases 44248 & 33716?

Reply:
good catch, removed the changes